### DISCUSSION TITLE
i18n HTML mark up in translations

### DISCUSSION THREAD

**Discussion by emin-alizada** [solidjs-community/solid-primitives#727] [Q&A] [answered]
I wonder if there is something similar to this:
https://next-intl.dev/docs/usage/messages#html-markup

```
{
  "markup": "This is <important>important</important>"
}
```

> // Returns 'This is <b>important</b>'
> t.markup('markup', {
>   important: (chunks) => `<b>${chunks}</b>`
> });

Since different languages can have different placement of the word. Also there might be more complicated cases like:

```
         <p>
           <button>
              Refresh the page
            </button>
            , the problem might be temporary. <br /> If the problem persist,{' '}
            <a
              href="#"
            >
              contact us
            </a>{' '}
            and we’ll help you get on your way. <br />
            Thanks for your patience!
          </p>
```

**Comment by atk** [comment, marked as answer]
Currently, there is not and it would be too complex to be part of the main primitive. Maybe an extension.

**Reply by emin-alizada** [reply]
Got it, thank you!
